src/share/export.js

```diff
@@ -0,0 +1,48 @@
+import fs from '../lib/fs';
+const sh = new fs.Shell();
+import WebTorrent from 'webtorrent';
+
+var client = new WebTorrent();
+const files = [];
+
+document.getElementById('btnSeed').addEventListener('click', startSeeding);
+console.log('inside of export.js!');
+
+function processPath(path, next) {
```

Comment:
Move this function inside `startSeeding` and then you don't need those global variables.